bravado/fido_client.py

```diff
@@ -114,6 +114,8 @@ def result(self, timeout=None):
         # type: (typing.Optional[float]) -> T
         try:
             return self._eventual_result.wait(timeout=timeout)
+        except fido.exceptions.HTTPTimeoutError:
+            raise  # Don't modify original exception
```

Comment:
Please add a comment (like in the PR description) explaining why you are catching the exception and re-raising it without performing actions.

Reply:
Done, let me know if it needs further clarification.

Reply:
Thanks a lot for doing it.
I've just approved your change.